content/getting-started/authorization/faq/index.mdx

```diff
@@ -0,0 +1,76 @@
+---
+weight: 40
+category: "getting-started"
+subcategory: "authorization"
+title: "Authorization FAQ"
+tagline: "Authorization-related frequently asked questions"
+desc: 'Authorization-related frequently asked questions'
+---
+
+# Getting started
+
+## What's the easiet way to authorize calls to the LiveChat API?
+For fast & easy authorization, we recommend **Personal Access Tokens (PATs)**. They're a way to go if you want to authorize private apps or speed up the development process.
+
+# General
+
+## Can I use the same access token to make calls to all APIs?
+No. Making calls to the Customer Chat API requires you to use a different token than for all other APIs. Read about [**Agent authorization flows**](/getting-started/authorization/#agent-authorization-flows)
+and [**Customer authorization flows**](/getting-started/authorization/#customer-authorization-flow).
+
+
+# Personal Access Tokens
+
+## How can I get a Personal Access Token?
+Log in to <a href="https://developers.livechatinc.com/console/" target="_blank" rel="noopener noreferrer">**Developer Console**</a> with your LiveChat login and password. Go to **Tools** and click **Create new token +**, or use an existing token if you have one.
+
+## Authorizing requests with a Personal Access Token doesn't work.
+1. Remember, PATs use the <a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication#Basic_authentication_scheme/" target="_blank">**Basic authentication scheme**</a>. Make sure you're not using something else, for example **Bearer**.
+2. Ensure, you're not using a PAT to authorize a call to the Customer Chat API. If that's what you want to achieve, see [**How can I make calls to the Customer Chat API?**](#how-can-i-make-calls-to-the-customer-chat-api)
+3. Check if your PAT is valid, complete, and has all necessary scopes. You can use our <a href="https://developers.livechatinc.com/console/tools/token-debugger/" target="_blank" rel="noopener noreferrer">**Token Debugger**</a>.  
+
+
+# Customer authorization 
+
+## How can I make calls to the Customer Chat API?
+You need to create a customer. You'll, then, use the access token of this customer to authorize your calls to the Customer Chat API. To create a customer, send a request described in [**Creating new customer**](/getting-started/authorization/#creating-new-customer).
```

Comment:
You'll, then, use the access - no commas needed I think? Or perhaps loose one :)